download_models/model_build/convert-model.py

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Do we need to add copyright headers to new files?

Reply:
added the copyright headers